src-graphql/images-query.js

```diff
@@ -0,0 +1,9 @@
+export default function imagesQuery(fields = ['id', 'src', 'altText']) {
```

Comment:
There's an imageQuery and imagesQuery because one is a connection (in products) and the other is singular (in collections). I thought about combining them within one function but that required an extra boolean to be passed with every query. Do you have any suggestions?